Cargo.toml

```diff
@@ -38,6 +38,7 @@ rkyv-validation = ["rkyv?/validation"]
 __internal_bench = []
 
 [dependencies]
+num-traits = { version = "0.2", default-features = false }
```

Comment:
Why is this being added?

Reply:
...??? Very odd merge conflict while rebasing changes.